lib/wine_o_inventory_api_web/views/api/wine_view.ex

```diff
@@ -0,0 +1,17 @@
+defmodule WineOInventoryApiWeb.WineView do
+    use WineOInventoryApiWeb, :view
+
+    def render("index.json", %{wines: wines}) do
+        render_many(wines, WineOInventoryApiWeb.WineView, "wine.json")
+    end
+
+    def render("wine.json", %{wine: wine}) do
```

Comment:
What does this do for getting a list of wines?

Reply:
From an example on the docs, it seems that under the hood, it's following something similar to `Enum.map`. `render_many` maps over each item in `wines` and calls `WineView.render("wine.json", %{wine: wine})`, which returns a `Map`. The result is a `List` of `Maps`